### ISSUE TITLE
xtb 6.3.3 segfaults on ip/ea calculations

### ISSUE THREAD

**Issue by githubgregersen** [grimme-lab/xtb#413]
Using:
```
xtb version 6.3.3 (5b13467) compiled by 'ehlert@majestix' on 2020-09-17
```
I get a segfault when running the command:
```
xtb --vipea -P 1 test.xyz
```

test.xyz is:
```
19
Example
   C    5.142600059509277   0.9739000201225281  -0.2004999965429306
   N   3.7163000106811523   0.8838000297546387  -0.2667999863624573
   C    2.911099910736084   0.8471999764442444  -1.3568999767303467
   C   1.6059000492095947   0.7591999769210815  -0.9039999842643738
   C   0.3352000117301941   0.6873000264167786  -1.5124000310897827
   C  -0.7645000219345093   0.6086999773979187  -0.7081999778747559
   C  -0.6639000177383423   0.5965999960899353   0.7020999789237976
   C   0.5471000075340271   0.6640999913215637   1.3315000534057617
   C    1.686900019645691   0.7462999820709229   0.5218999981880188
   N   2.9858999252319336    0.823199987411499   0.8705999851226807
   O    3.527899980545044   0.8452000021934509   2.0004000663757324
   H   5.3968000411987305   1.0104000568389893   0.8622000217437744
   H    5.609000205993652   0.1005999967455864  -0.6575000286102295
   H    5.496399879455566   1.8782999515533447  -0.6955000162124634
   H   3.3132998943328857   0.8840000033378601   -2.347399950027466
   H   0.2513999938964844   0.6947000026702881  -2.5882999897003174
   H  -1.7479000091552734   0.5533000230789185  -1.1513999700546265
   H  -1.5700000524520874   0.5320000052452087     1.28410005569458
   H   0.6499999761581421   0.6567000150680542    2.404599905014038
```

The end of the output when using --verbose shows:
```
           -------------------------------------------------
          |                 I P E A - x T B                 |
           -------------------------------------------------

        Reference                      10.1039/c7sc00601b
      * Hamiltonian:
        H0-scaling (s, p, d)           2.150000    2.300000    2.000000
        zeta-weighting                 0.000000
      * Dispersion:
        s8                             2.400000
        a1                             0.630000
        a2                             5.000000
        s9                             0.000000
      * Repulsion:
        kExp                           1.500000
        rExp                           1.000000
      * Coulomb:
        alpha                          1.000000
        third order                    atomic
        anisotropic                    false
      * Halogen bond correction:
        rad-scale                      1.300000
        damping                        0.440000

forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source             
xtb                000000000326DC03  Unknown               Unknown  Unknown
xtb                000000000348DFB0  Unknown               Unknown  Unknown
xtb                000000000086C7AB  xtb_scc_core_mp_q        1326  scc_core.f90
xtb                000000000087D1B9  xtb_scf_mp_scf_.V         285  scf_module.f90
xtb                00000000005B3A6F  xtb_xtb_calculato         157  calculator.f90
xtb                00000000008C7819  xtb_single_mp_sin          63  single.f90
xtb                000000000040F5EB  xtb_prog_main_mp_         577  main.F90
xtb                0000000000439509  MAIN__                     55  primary.f90
xtb                00000000004049D2  Unknown               Unknown  Unknown
xtb                000000000348F450  Unknown               Unknown  Unknown
xtb                00000000004048B7  Unknown               Unknown  Unknown
```

**Comment by githubgregersen** [comment]
Hmm. I just saw #399, maybe this is fixed in src

**Comment by awvwgk** [comment]
This should be fixed by #399 on the current master branch and should be fixed in the [continuous release version](https://github.com/grimme-lab/xtb/releases/tag/bleed).

**Comment by awvwgk** [comment]
Closing this as already resolved
